README.md

```diff
@@ -159,8 +187,18 @@ This command will git-commit files in `docs/elpa/` (or the directory
 | `-t, --tar <tar-executable>`      | (Use value from `package-build.el`) | Specify tar executable name to archive files |
 
 
+Example
+-------
+
+- [JCS-ELPA](https://github.com/jcs-emacs/jcs-elpa)
```

Comment:
Sorry I don't want to add this section, because I don't want to receive PRs to add new repositories here...

Reply:
No problem! I will remove it!

I thought it would be nice to have at least some examples since many users asked me how to use github-elpa. 🤔

Reply:
Thanks 🙌 

> I thought it would be nice to have at least some examples since many users asked me how to use github-elpa. 🤔

It is true, I hope users will find examples by searching GitHub instead...